tests/load_from_macro/BUILD

```diff
@@ -1,3 +1,5 @@
+load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
+
 # Copyright 2019 The Bazel Authors. All rights reserved.
```

Comment:
This is only used for tests, do we really need this (especially as public)?

Reply:
https://github.com/bazelbuild/rules_proto/pull/66